samples/bluetooth/fast_pair/locator_tag/include/app_fp_adv.h

```diff
@@ -32,11 +32,29 @@ enum app_fp_adv_mode {
 	APP_FP_ADV_MODE_NOT_DISCOVERABLE,
 };
 
-/** Set the Fast Pair advertising mode.
+/** Register a trigger for Fast Pair advertising.
  *
- * @param adv_mode Fast Pair advertising mode.
+ *  @param trigger Trigger identifier.
+ *  @return 0 if the operation was successful. Otherwise, a (negative) error code is returned.
+ */
+int app_fp_adv_trigger_register(const char *trigger);
```

Comment:
I would suggest to inform in API doc that the same pointer must be used for all the time by a subscriber (we do not perform strcmp). Pointer validity must be ensured too